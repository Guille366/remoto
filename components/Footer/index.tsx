import { memo } from "react";

const Footer: React.FC = () => {
  const date = new Date();
  const year = date.getFullYear();

  return (
    <footer className="flex items-center justify-center w-full h-14 font-mono px-4">
      <div className="w-full flex flex-row justify-between items-center m-auto h-14 max-w-screen-lg border-t border-gray-200">
        <p className="text-start text-gray-500 text-xs">
          &copy;{year} <span className="font-bold text-gray-500">REMOTO</span>.
          Todos os direitos reservados.
        </p>
        <p className="text-end text-gray-400 text-xs">
          by{" "}
          <a
            href="https://github.com/vorsakha"
            target="_blank"
            rel="noopener noreferrer"
          >
            @vorsakha
          </a>
        </p>
      </div>
    </footer>
  );
};

export default memo(Footer);
