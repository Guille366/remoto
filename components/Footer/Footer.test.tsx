import { render } from "@testing-library/react";
import Footer from ".";

jest.useFakeTimers('modern').setSystemTime(new Date(2022, 9, 1, 7));

describe('Loading', () => {
  it('renders properly', () => {
    const { container } = render(<Footer />);

    expect(container.firstChild).toMatchSnapshot();
  });
});