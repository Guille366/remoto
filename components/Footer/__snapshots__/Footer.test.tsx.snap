// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders properly 1`] = `
<footer
  class="font-nunito flex items-center justify-center w-full h-14 px-4 text-xs"
>
  <div
    class="w-full flex flex-row justify-between items-center m-auto h-14 max-w-screen-lg border-t border-gray-200"
  >
    <p
      class="text-start text-gray-500"
    >
      Â©
      2022
       
      <span
        class="font-code"
      >
        REMOTO
      </span>
      .
    </p>
    <p
      class="text-end text-gray-400"
    >
      by
       
      <a
        class="font-code"
        href="https://github.com/vorsakha"
        rel="noopener noreferrer"
        target="_blank"
      >
        @vorsakha
      </a>
    </p>
  </div>
</footer>
`;
